singularities/samtools/samtools-1.20.def

```diff
@@ -0,0 +1,23 @@
+Bootstrap: docker
+
+# The build-stage image:
+From: continuumio/miniconda3:24.4.0-0
+
+%files
+	/home/xberfa/scripts/build_sing/samtools-1.20.yml /dependencies/samtools-1.20.yml
+    /apps/bio/dependencies/wgs_somatic/hg38/sentieon/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna /references/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna
+    /apps/bio/dependencies/wgs_somatic/hg38/sentieon/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.fai /references/GCA_000001405.15_GRCh38_no_alt_analysis_set.fna.fai
```

Comment:
Should these be here? Why are they not input arguments to the singularity when running the snakemake rule? Requires a bit of binding but that shouldn't be too difficult to manage

Reply:
Removed, using the reference located on the cluster instead. Specified in the cluster config.